app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -416,14 +432,68 @@ private void showFeedInformation(final Feed feed, Map<String, String> alternateF
 
         viewBinding.backgroundImage.setColorFilter(new LightingColorFilter(0xff828282, 0x000000));
 
-        View header = View.inflate(this, R.layout.onlinefeedview_header, null);
-
-        viewBinding.listView.addHeaderView(header);
+        viewBinding.listView.addHeaderView(headerBinding.getRoot());
         viewBinding.listView.setSelector(android.R.color.transparent);
         viewBinding.listView.setAdapter(new FeedItemlistDescriptionAdapter(this, 0, feed.getItems()));
 
-        TextView description = header.findViewById(R.id.txtvDescription);
+        final int characterLimit = 165;
+        boolean isLong = false;
 
+
+        headerBinding.txtvDescription.setMaxEms(characterLimit);
```

Comment:
Doesn't that possibly conflict with the span you are adding?

Reply:
Nope, it's working fine.